src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -28,9 +28,6 @@
 #include "ci/ciEnv.hpp"
 #include "ci/ciUtilities.hpp"
 #include "code/compiledIC.hpp"
-#if INCLUDE_CDS
```

Comment:
Did you mean to only remove `INCLUDE_CDS` , and add the `#include` for `autoCodeCache.hpp`?

Reply:
`macroAssembler_aarch64.hpp` already has #include.